src/backend/executor/nodeBitmapHeapscan.c

```diff
@@ -73,6 +73,20 @@ static inline void BitmapPrefetch(BitmapHeapScanState *node,
 static bool BitmapShouldInitializeSharedState(ParallelBitmapHeapState *pstate);
 static void ExecEagerFreeBitmapHeapScan(BitmapHeapScanState *node);
 
+/*
+ * Other non-heap table access method may use bitmap scan,
+ * the prefetch will not work for them if they have
+ * non-standard page-based storage.
+ */
+static inline bool
+RelationSupportPrefetch(Relation rel)
+{
+	bool non_standard;
+	non_standard = (rel->rd_rel->relkind == RELKIND_RELATION ||
+					rel->rd_rel->relkind == RELKIND_MATVIEW) &&
+					!RelationIsHeap(rel);
```

Comment:
non_standard will be false here if relkind is not RELKIND_RELATION, that support prefresh.Is it right?

Reply:
Bitmap scan will not scan other type of relkind, right?

Reply:
Then it seems we don't need to judege relkind here.

Reply:
Has removed. Thanks for your suggestion.